src/FormBuilder.php

```diff
@@ -429,6 +446,66 @@ private function buildHtmlAttrs(array $attributes, $appendAttrs = true): string
         ));
     }
 
+    private function customClass(string $element = 'input')
+    {
+    	extract($this->get('type', 'size', 'inline', 'render'));
+
+    	$type = $type ?? $render;
+    	$input = $element === 'input';
+	    $wrapper = $element === 'wrapper';
+    	$label = $element === 'label';
+
+	    if($input){
+	    	$class = 'custom-control-input';
+	    } elseif($wrapper){
+	    	$class = 'custom-control' . (isset($inline) && $inline ? ' custom-control-inline' : '');
+	    } else {
+	    	$class = 'custom-control-label';
+	    }
+
+	    switch($type){
+		    case 'checkbox':
+			    if($wrapper){
+				    $class .= ' custom-checkbox';
+			    }
+			    break;
+		    case 'radio':
+			    if($wrapper){
+				    $class .= ' custom-radio';
+			    }
+			    break;
+		    case 'switch':
+			    if($wrapper){
+				    $class .= ' custom-switch';
+			    }
+			    break;
+		    case 'select':
+			    if($input){
+				    $class = 'custom-select';
+				    $class = $size ? $class . '-' . $size : $class;
+			    } elseif($label || $wrapper){
+			    	$class = '';
+			    }
+			    break;
+		    case 'range':
+			    $class = $input ? 'custom-range' : '';
+			    break;
+		    case 'file':
+		    	if($wrapper) {
+		    		$class = 'custom-file';
+			    } elseif($input) {
+		    		$class = 'custom-file-input';
+			    } else{
+		    	    $class = 'custom-file-label';
+			    }
+			    break;
+		    default:
+		    	// do nothing
+			    dd('type', $type);
```

Comment:
This dd() debug function should be removed

Reply:
How did I miss that? It's done.